### PR TITLE
Added some bacon humor to pig farm story

### PR DESCRIPTION
**Pull request by jkenney9a**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1693)

### CODE REVIEW

File: english/water-dreams/pig-farm/pig-farm.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ Before long you hear a snort, and a large snout invades the traquil spaces of di
 that make up the pigs meal. You feel yourself being sucked into his mouth, pulled down into his stomach...
 and suddenly you have eyes, and feet and sharp tusks at the end of your face.
 
+As the realization that you have become a pig winds its way through your
+now swine infused brain, you think...
+
+["Wow, I still love bacon...but now I hate myself!"](/english/water-dreams/pig-farm/love-bacon/love-bacon.md)
```
Comment (casey-collab) [inline english/water-dreams/pig-farm/pig-farm.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/water-dreams/pig-farm/pig-farm.md [hunk 2]
```diff
@@ -6,3 +6,9 @@ Before long you hear a snort, and a large snout invades the traquil spaces of di
...
+
+["I can't believe I ever loved bacon! I must make amends!"](/english/water-dreams/pig-farm/hate-bacon/hate-bacon.md)
```
Comment (casey-collab) [inline english/water-dreams/pig-farm/pig-farm.md]: This link is also invalid.

File: english/water-dreams/pig-farm/hate-bacon/hate-bacon.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+You've never felt so alive in your life! Your purpose has become
```
Comment (casey-collab) [inline english/water-dreams/pig-farm/hate-bacon/hate-bacon.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/water-dreams/pig-farm/love-bacon/love-bacon.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+You take a moment to reflect and realize your psychological state is
```
Comment (casey-collab) [inline english/water-dreams/pig-farm/love-bacon/love-bacon.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Sorry, Casey doesn't understand that kind of link. This looks good to me! :pig2:

**Comment by jkenney9a** [comment]
Ah, apologies for the confusion...and thanks for a great intro to github course Caroline!

**Pull request merged**